src/Features/CSharp/Portable/ConvertProgram/ConvertProgramHelpers.cs

```diff
@@ -67,15 +68,18 @@ public static async Task<Document> ConvertToProgramMainAsync(Document document,
             var hasExistingPart = programType.DeclaringSyntaxReferences.Any(d => d.GetSyntax(cancellationToken) is TypeDeclarationSyntax);
 
             var root = (CompilationUnitSyntax)await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+
+            var method = (MethodDeclarationSyntax)generator.MethodDeclaration(mainMethod, "Main", GetStatements(root));
```

Comment:
Not important. I'd expect **zero** customer report for this. But in case of a single statement, should we respect block vs expression body preference?

I'm expecting a "No" for this very minor scenario. But noting anyway as it came to mind :)

Reply:
Yeah, my caring level is zero.